lib/assets/javascripts/cartodb/table/tableview.js

```diff
@@ -311,6 +312,12 @@
         }
       },
 
+      _dismissSQLHeader: function (e) {
+        this.killEvent(e);
+        this.$('thead').find('.sqlview').remove();
+        this.$('thead > tr').css('height', '');
```

Comment:
I don't _think_ so...but I could very easily be missing something. All the _clearView does is call through to clearSQLView which actually resets the sql for the layer which we don't want to do in this case. I'm not sure exactly how that banner gets cleared in the case of _clearView -- I think table view detects the sql change and just calls _renderSQLHeader again